src/Microsoft.AspNetCore.Mvc.Abstractions/Microsoft.AspNetCore.Mvc.Abstractions.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <Import Project="..\..\build\common.props" />
+  <Import Project="..\..\build\sourcelink.props" />
```

Comment:
I tried the above and got an error about metadata not being allowed in the position. I got a similar error when I tried:
``` xml
<Import Project="..\sourcelink.props" Condition="'%(ProjectsToPack)' == '$(MSBuildProjectFullPath)'" />
```
Programming in xml is so efficient. :snail: